addons/sale/models/sale_order.py

```diff
@@ -1276,7 +1276,7 @@ def _cron_send_pending_emails(self):
         pending_email_orders = self.search([('pending_email_template_id', '!=', False)])
         self.env['ir.cron']._commit_progress(remaining=len(pending_email_orders))
         for order in pending_email_orders:
-            order = order[0]  # Avoid pre-fetching after each cache invalidation due to committing.
+            order = order.with_prefetch()  # Avoid pre-fetching after each cache invalidation due to committing.
```

Comment:
While we're at it, wouldn't it make sense to introduce a proxy method `without_prefetch()` that would call `with_prefetch(prefetch_ids=None)`?

I think this would make more sense and be more understandable in business code, since `with_prefetch()` actually disables pre-fetching.

Reply:
`with_prefetch` sets `prefetch_ids = ids` by default (breaking it). Setting it to None is more confusing to me.